src/WorkBC.Data/Migrations/20240415225803_AddFKNoCCode2021.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace WorkBC.Data.Migrations
+{
+    public partial class AddFKNoCCode2021 : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            //Add new columns to dbo.Jobs and dbo.JobVersions tables.
+            migrationBuilder.AddColumn<string>(
```

Comment:
I was expecting to see a foreign key relationship to the `NocCodes2021` table.

Reply:
It's done in the latest commit.